doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1543,6 +1544,16 @@ By default this setting is absent and the Dataverse Software assumes it to be fa
 
 ``curl -X PUT -d 'true' http://localhost:8080/api/admin/settings/:IndependentHandleService``
 
+.. _:HandleAuthHandle:
+
+:HandleAuthHandle
++++++++++++++++++++++++++
+
+Specific for Handle PIDs. Set this setting to <prefix/suffix> to be used on a global handle service when the public key is NOT stored in the default handle.
+By default this setting is absent and the Dataverse Software assumes it to be not set.
+
+``curl -X PUT -d '<prefix/<suffix>' http://localhost:8080/api/admin/settings/:HandleAuthHandle``
+
```

Comment:
just an inconsistency with the '<prefix/suffix>' in the description/example. please remove the "<" in the example

Reply:
HI,

I modified the documentation to be more clear with a real world example.
A new commit was done..

Greetings..